src/high-level/include/iDynTree/KinDynComputations.h

```diff
@@ -1178,6 +1178,42 @@ class KinDynComputations {
                          const LinkNetExternalWrenches & linkExtForces,
                                FreeFloatingGeneralizedTorques & baseForceAndJointTorques);
 
+    /**
+     * This method is similar to inverseDynamics, but provides as an additional output the internal 6D force/torques (aka wrenches)
+     * excerted by the two links connected to each joint, in the linkInternalWrenches argument.
+     *
+     * The `linkInternalWrenches` is a container of \f$n_{L}\f$ (number of links) 6D Force/Torques, one associated to each link.
+     * In particular, if the link \f$L\f$ is the link with index \f$L\f$ the element linkInternalWrenches(i) contains, depending on the
+     * choice of `FrameVelocityRepresentation`:
+     *
+     * |`FrameVelocityRepresentation` |  `linkInternalWrenches(i)` |
+     * |:----------------------------:|:--------------------------:|
+     * | `MIXED_REPRESENTATION` (default) | \f$ {}_{L[A]} \mathrm{f}_{\lambda(L), L} \f$ |
+     * | `BODY_FIXED_REPRESENTATION` | f$ {}_{L} \mathrm{f}_{\lambda(L), L} \f$ |
+     * | `INERTIAL_FIXED_REPRESENTATION` | \f$ {}_{A} \mathrm{f}_{\lambda(L), L} \f$ |
+     *
+     * Where if $C$ is a given frame, \f$ {}_{C} \mathrm{f}_{\lambda(L), L} \f$ is the 6D force/torque that the
+     * parent link \f$\lambda(L)\f$ excerts on its child \f$L\f$.
```

Comment:
do u mean:

Where if $C$ is a given frame, \f$ {}_{C} \mathrm{f}_{\lambda(L), L} \f$ is the 6D force/torque that the parent link \f$\lambda(L)\f$ excerts on its child \f$L\f$ represented in frame $C$?

Reply:
Thanks, given the PR was already merged I did this modification in https://github.com/robotology/idyntree/pull/858 . Feel free to comment other parts that you do not understand, thanks!